### ISSUE TITLE
[Accessibility] p-dropdown information is only announced after pressing ctrl+down arrow twice. 

### ISSUE THREAD

**Issue by t-dazilb** [primefaces/primeng#10156]
Current Queue Time for Review
Without PRO Support: ~8-12 weeks.
With PRO Support: 1 hour

**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please demonstrate your case at stackblitz by using the issue template below. Issues without a test case have much less possibility to be reviewd in detail and assisted.

https://stackblitz.com/edit/github-ypgqnx?file=src/app/app.component.ts

**Current behavior**
In scan mode, extra navigation steps appear while navigating dropdown (Screen reader announces information about the dropdown only after pressing down key twice, and the announced message does not indicate that it's a drop down).
Issue can also observe the issue in the demo page.

**Expected behavior**
The narrator should announce dropdown details on initial navigation ( as described in the narrator documentation: https://support.microsoft.com/en-us/windows/chapter-3-using-scan-mode-7b2af804-5a2f-90fd-b0e0-672f7cbbf2da

**Minimal reproduction of the problem with instructions**
Goto https://www.primefaces.org/primeng/#/dropdown or https://stackblitz.com/edit/github-ypgqnx?file=src/app/app.component.ts.
Run narrator in scan mode (Caps Lock + pace). Navigate to the dropdown using Ctrl+down arrow key.
Notice that the narrator announces the dropdown's details after pressing Ctrl+down twice.

**What is the motivation / use case for changing the behavior?**
Users who rely on assistive technology would be confused as the screen reader does not read the information related to the dropdown on the first press of down arrow key.

**Please tell us about your environment:**
Windows 10 + Windows narrator

* **Angular version:** 11.2.0

* **PrimeNG version:** 9.1.3
(Issue also presents in latest versions) 

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by bdhanoa** [comment]
@yigitfindikli, are you planning to correct this issue soon?

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
